index.html

```diff
@@ -41,6 +41,191 @@ <h1 class="primary-heading"> I’m Mariane Glad to see you! </h1>
 
         </ul>
     </section>
+
+    <section class="Projects">
```

Comment:
I suggest you ensure this section follows [project instructions]() and uses `display:grid`. The section should have use `display:grid` and you can let the individual cards/articles remain with `display:flex`
The screenshot below illustrates this instruction:
<img width="1666" alt="Screenshot 2023-01-13 at 18 30 32" src="https://user-images.githubusercontent.com/48546992/212357781-91343f6e-34e5-49de-aa33-2b5f24e6715d.png">